src/xpanse-api/generated/models/ServiceDetailVo.ts

```diff
@@ -111,5 +111,8 @@ export namespace ServiceDetailVo {
         DESTROYING = 'DESTROYING',
         DESTROY_SUCCESS = 'DESTROY_SUCCESS',
         DESTROY_FAILED = 'DESTROY_FAILED',
+        MIGRATING = 'MIGRATING',
```

Comment:
these files must not be touched. These are auto generated from the API. 
Basically these values will never come in the API response.